cleanup-passes/travis.ts

```diff
@@ -32,56 +31,133 @@ async function cleanupTravisConfig(element: ElementRepo): Promise<void> {
 
   let travis = yaml.safeLoad(travisConfigBlob);
 
-  let needsReview = false;
+  // a small state class to represent the decision to make a PR or a commit to head
+  // Once tripped, a PR will be made
+  class Reviewer {
+    private _needsReview = false
+    get needsReview() {
```

Comment:
It'd be nice for this logic to live on ElementRepo. We could pretty easily turn that into a class.

Mind doing that here? If not, leave a comment indicating that we should do that.